nucypher/cli/characters/ursula.py

```diff
@@ -67,11 +66,6 @@
 @click.option('--provider-uri', help="Blockchain provider's URI", type=click.STRING)
```

Comment:
Maybe we can write a format example (such as "ipc:///path/to/geth.ipc"), otherwise it could be unclear what fails when, in fact, it's just not enough slashes

Reply:
:heavy_check_mark: